examples/applications/simulated-camera/src/camera.rs

```diff
@@ -123,9 +123,9 @@ impl CameraLogic {
     }
 }
 
-// In 1.64 clippy checks get_first
```

Comment:
Was removal of this intentional? Does it still apply?

Reply:
Yes, it still applies and was also intentional. The updated comment:

```
// try_stream macro is still using accessing first element with
// `$crate::async_stream_impl::try_stream_inner!(($crate) $($tt)*).get(0)`
// https://rust-lang.github.io/rust-clippy/master/index.html#get_first
```

contains the same information as before, but no longer mentions with which version this was introduced. Otherwise, I would have needed to explicitly mention `1.64` and `1.65` as it applies to both versions, instead I future-proofed it by removing any mention of a version.

Reply:
> Otherwise, I would have needed to explicitly mention `1.64` and `1.65` as it applies to both versions, instead I future-proofed it by removing any mention of a version.

Understandable, which is why I was expecting something like _1.64 or later_ or just _1.64+_.

Reply:
Valid point, I pushed d7d9b619e16dc83185b6d91b52357b637eea61e6 to have a less noisy change